<launch>
    <!-- 参数定义 -->
    <!-- PID参数（仿真使用) -->
    <param name="controller/linear_kp" value="1" />
    <param name="controller/linear_ki" value="0" />
    <param name="controller/linear_kd" value="0.0" />
    <param name="controller/angular_kp" value="8" />
    <param name="controller/angular_ki" value="0.0" />
    <param name="controller/angular_kd" value="0.5" />
    <!-- PID参数（实机使用) -->
    <!-- <param name="controller/linear_kp" value="0.6" />
    <param name="controller/linear_ki" value="0" />
    <param name="controller/linear_kd" value="0.0" />
    <param name="controller/angular_kp" value="2.5" />
    <param name="controller/angular_ki" value="0.0" />
    <param name="controller/angular_kd" value="0.2" /> -->
    <param name="use_simulation" value="true" />
    <!-- 启动的节点 -->
    <group ns="robot_1">
        <node pkg="vel_modulation" type="multi_control_node" name="control_node" output="screen"/>
    </group>
    <group ns="robot_2">
        <node pkg="vel_modulation" type="multi_control_node" name="control_node" output="screen"/>
    </group>
    <group ns="robot_3">
        <node pkg="vel_modulation" type="multi_control_node" name="control_node" output="screen"/>
    </group>
</launch>